### ISSUE TITLE
Automated releases

### ISSUE THREAD

**Issue by peektism** [OpenZeppelin/polkadot-runtime-templates#70]
### What is the feature you would like to see?

Follow up to #30 to automate the release process by tracking changes in CHANGELOG.md and issuing a release depending on changes recorded therein

See https://github.com/polkadot-fellows/runtimes/pull/63 for reference

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines

**Comment by ozgunozerk** [comment]
Blocked by https://github.com/MarcoIeni/release-plz/issues/1144

**Comment by ozgunozerk** [comment]
Yes it is still blocked, I was contributing to the PR in the upstream to make it faster, but seems like that PR is stopped. I might take another look when I have time. But afaik, this is not a priority for us and currently we have more pressing issues. In short, yes, let's keep it in backlog for now
